plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TrinoCatalogFactory.java

```diff
@@ -39,6 +39,7 @@
     private final CatalogType catalogType;
     private final boolean isUniqueTableLocation;
     private final boolean isUsingSystemSecurity;
+    private final String warehouse;
```

Comment:
This should be named more meaningfully.  Let's figure config property name first.

Reply:
sure, I updated to `catalogWarehouse` for now